components/slider/index.css

```diff
@@ -10,63 +10,94 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
+
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
```

Comment:
Have we checked with design to see if the tick marks are being removed? I don't see them represented in the XD redline or at https://spectrum.adobe.com/page/slider/

If we are keeping them, there may be a way to calculate their height relative to the controls height.